tests/unit/test_gff3_parser.py

```diff
@@ -0,0 +1,102 @@
+"""
+Data-driven tests for rna quantification.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import ga4gh.gff3 as gff3
+import unittest
+
+
+class TestGff3ParserOnTypicalFile(unittest.TestCase):
+    """
+    Data driven unit tests for the GFF3 parser
+    """
+    def setUp(self):
```

Comment:
It's not only BioPython, it's a also an add-on dependency that
is indented to become part of BioPython but has been stagnant
for a year:

  https://pypi.python.org/pypi/bcbio-gff/0.6.2

requiring two parsers is a rather heavy weight way to have
expected results.

Jerome Kelleher <notifications@github.com> writes:
> Well, we're talking about a develop-time dependency here, not a run-time
> dependency. I don't think BioPython is that scary a develop-time dependency.
> Once it doesn't murder our CI builds, I think it would probably be OK.
> 
> —
> Reply to this email directly or view it on GitHub.*
>